src/modules/configurator/configure-a-car/CarSelect.tsx

```diff
@@ -59,14 +60,14 @@ export function CarSelect() {
             scrollbar={{ draggable: true }}
           >
             {carsArray &&
-              carsArray.map((car: Car, index) => (
+              carsArray.map((car: CarConfigurationsWithId, index) => (
                 <SwiperSlide key={index}>
                   <CarItem
                     key={index}
                     id={car.id}
-                    model={car.name}
+                    model={car.model}
```

Comment:
If the has the same props CarConfigurationsWithId, than you can just spread the car object `{...car}`

Reply:
Passed everything as `{...car}`.